Upgrade to Composable Kernel 7.2 (#396)
amd/rocm-examples: Libraries/ComposableKernel/gemm/gemm/README.md

## Description

> This example demonstrates how to perform a GEMM operation with the CK Tile

Reviewer: ```suggestion
This example demonstrates how to perform a GEMM operation using CK Tile
```

> programming model. Five variants are shown:

Reviewer: ```suggestion
. Five variants are shown:
```
